packages/nuxt/src/core/nuxt.ts

```diff
@@ -37,7 +37,17 @@ export function createNuxt (options: NuxtOptions): Nuxt {
     hook: hooks.hook,
     ready: () => initNuxt(nuxt),
     close: () => Promise.resolve(hooks.callHook('close', nuxt)),
-    vfs: {}
+    vfs: {},
+    app: {
```

Comment:
For future compatibility with multi-layer/apps, i would suggest to expose it as `nuxt.apps.default`

Reply:
:+1: